### PR TITLE
Adding configuration table

### PR DESCRIPTION
**Pull request by vds**

### CODE REVIEW

File: golang-base/postgres/postgres_notary.sql [hunk 1]
```diff
@@ -1,5 +1,15 @@
 -- Setup ran by the notary_test user. Setup schema etc.
 
+CREATE TABLE IF NOT EXISTS configuration (
+       address text,
+       port text, 
+       private_key bytea,
+       public_key bytea
+       );
```
Comment (teknico) [inline golang-base/postgres/postgres_notary.sql]: Can we align this to the left?

**Comment by toxeus** [comment]
needs a rebase

**Comment by toxeus** [comment]
:+1: but you probably still want to align the parentheses as teknico suggested ;)

**Comment by vds** [comment]
Fixed parens

**Comment by toxeus** [comment]
Again :+1:

File: golang-base/postgres/postgres_notary.sql [hunk 2]
```diff
@@ -1,5 +1,15 @@
 -- Setup ran by the notary_test user. Setup schema etc.
 
+CREATE TABLE IF NOT EXISTS configuration (
+       address text,
```
Comment (benma) [inline golang-base/postgres/postgres_notary.sql]: Isn't this normally called `host`?

File: golang-base/postgres/postgres_notary.sql [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+       port text,
```
Comment (benma) [inline golang-base/postgres/postgres_notary.sql]: Why `text` and not `int`?

**Comment by teknico** [comment]
Seconding @benma's points. Actually a hostname has a stricter syntax than just text, not sure if we want to enforce that at the database level.

Also, the port number is a `uint16`. Again, not sure about the level of precision in the numeric datatype that we want.

**Comment by benma** [comment]
@teknico as precise as possible, I suppose.

**Comment by vds** [comment]
It will be used as string in the code, what's the point of having an in and converting it all the time?

**Comment by benma** [comment]
@vds 

"127.0.0.1:hello world" 
Types should always be as specific as possible, so you have to do less manual sanity checking.

**Comment by vds** [comment]
"127.0.0.1:2147483647" is just as invalid. In postgresql "smallint" is too small (-32768 to +32767), "integer" is too big (-2147483648 to +2147483647).

**Comment by vds** [comment]
I'll use an integer and validate it when starting the server.

**Pull request closed without merging**